README.md

```diff
@@ -54,25 +54,27 @@ To uninstall the chart:
 | deployment.revisionHistoryLimit          | The number of old history to retain to allow rollback                                                                                      | `2`            |
 | deployment.command                       | Command for primary container of deployment                                                                                                | `[]`           |
 | deployment.args                          | Arg for primary container of deployment                                                                                                    | `[]`           |
+| deployment.priorityClassName             | Defines the priority class for pod scheduling                                                                                              | `""`           |
```

Comment:
`priorityClassName` is missing in the CronJob section of the documentation.

Reply:
@aslafy-z 
right, done